src/components/Patient/Utils.ts

```diff
@@ -17,3 +23,18 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+
+export const useAddPatientNote = (options: { patientId: string }) => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: mutate(routes.addPatientNote, {
+      pathParams: { patientId: options.patientId },
+    }),
+    onSuccess: (data) => {
+      queryClient.invalidateQueries({ queryKey: ["notes"] });
```

Comment:
```suggestion
      queryClient.invalidateQueries({ queryKey: ["notes", patientId] });
```

Reply:
When adding patientId it's not working what should i do now?

Reply:
define "not working"